src/index.html

```diff
@@ -1,12 +1,364 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="main-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="main-page__content">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a> 
+      <nav class="header__nav nav">
```

Comment:
```suggestion
      <nav class="header__nav nav">
```

Reply:
(Cheklist 9. [BEM] - create a separate file per each BEM block styles that have the same name as the block.) As for me, you should create a separate file for this block.

Reply:
but I do have the header block, isn't it enough?

Reply:
In html it is absolutely okay. But you really need separate file for styles of this block